src/Components/ListsItem.js

```diff
@@ -1,16 +1,17 @@
 import { List } from "@mui/material";
 
 import Listitem from "./Listitem";
-const ListsItem = (props) => {
+const ListsItem = ({ tasks, handleEditClick, handleDeleteItem }) => {
+  console.log(tasks);
```

Comment:
always check your code. so there is no console.log

Reply:
sorry sir i will change it